tasks/read_record.rb

```diff
@@ -0,0 +1,38 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require '../lib/service_now_request.rb'
+
+# Return an error
+def return_error(message)
+  result = {}
+  result[:_error] = {
+    msg: message,
+    kind: 'servicenow_tasks.read_record',
+    details: {},
+  }
+  # puts result.to_json
+  raise TaskHelper::Error.new('Failure!', 'snow_record.read_record', result)
+end
+
+# This task reads incidents
+class ServiceNowRead < TaskHelper
```

Comment:
This could be a nit, but I think `read_record` => `get_record`. I think of `read_<thing>` as "read `<thing>`'s content". Records don't have "content" (not like files). They're just a JSON of fields, which is what this is returning, and that "JSON of fields" is better captured by `get_record`.

Reply:
Changed name to get_record.